marzipan.py

```diff
@@ -0,0 +1,128 @@
+import pyone
+import configparser
+import os
+import time
+
+class one_interface(object):
+
+    def __init__(self):
+        self.config = None
+        self.client = None
+        self.root_public_key = None
+        self.template = None
+        self.template_id = None
+        self.cluster_vm_ids = []
+
+    def get_config(self):
+        """
+        Read configuration options from file
+        """
+        Config = configparser.ConfigParser()
+        Config.read('ClusterConf.ini')
+        self.config = Config
+
+    def get_one_server(self):
+        """
+        Connect to OpenNebula server
+        """
+        uname = self.config['one']['username']
+        pwd = self.config['one']['password']
+        endpoint = self.config['one']['endpoint']
+        one_server = pyone.OneServer(endpoint, session = uname+':'+pwd)
+        self.client = one_server
```

Comment:
It is true that this is the client to connect to the open nebula end point, but maybe it's confusing to call client the instance of the server class?

Reply:
@fnattino yes, good point. how would you feel about `one_client` ?

Reply:
sounds good, I think it would be much clearer!